index.html

```diff
@@ -2838,24 +2815,17 @@ <h3><dfn>Writing or pushing content</dfn></h3>
     </section>
 
     <section><h3>Validating external type</h3>
-      <p>
+      <div>
         The [[NFC-RTD]] specifies that external types MUST contain
-        the domain name of the issuing organization, a colon and a type
-        name that is at least one character long.
-      </p>
-      <p>
-        The [[NFC-RTD]] specifies to omit the URN prefix “`urn:nfc:ext:`”
-        in the binary encoding of external type records, therefore
-        applications SHOULD NOT specify that when creating
-        <a>external type records</a>.
-      </p>
+        the domain name of the issuing organization, a colon `U+003A` (`:`)
+        and a type name that is at least one character long, for instance
+        "`w3.org:member`".
+      </div>
       <p>
-        When the domain name is not provided, by default Web NFC uses the
-        <a data-cite="html">effective domain</a> of the {{Document/origin}} as
-        domain name in external type names when
-        <a href="#mapping-external-data-to-ndef">
-        creating external type records</a>. Therefore, applications MAY specify
-        only the colon and the external type name.
+        The [[NFC-RTD]] specifies the URN prefix “`urn:nfc:ext:`” as well, but
+        it is not written, i.e. it is not used in the binary encoding of
```

Comment:
is not stored in the NDEF record,

Reply:
should we keep the other part, i.e. use

> but is not stored in the NDEF record, i.e. it is not used in the binary encoding of  
<a>external type records</a>,

or should we just say

> but is not stored in the NDEF record, therefore Web NFC applications ...

Reply:
I chose the second.